src/main/steps/applicant1/hub-page/joint/jointTemplateSelector.ts

```diff
@@ -40,7 +40,6 @@ export const getJointHubTemplate = (
     case State.AwaitingAdminClarification:
     case State.AwaitingLegalAdvisorReferral:
       return HubTemplate.AwaitingLegalAdvisorReferral;
-    case State.FinalOrderOverdue:
```

Comment:
i wonder if this will break existing functionality. i.e. app1 applies -> overdue, app1 doesnt seem this page now as its in awaitingjointfinalorder.

Reply:
I think that's correct. If app1 has done what they need to, then they shouldn't get any more hassle about it being late